client/src/redux/actions/observations.js

```diff
@@ -22,6 +29,23 @@ export const addObservation = (userSession, observation) => async dispatch => {
   });
 };
 
+export const deleteObservations = userSession => async dispatch => {
+  const obs = await getObservations(userSession);
+  if (obs) {
+    const obsArray = JSON.parse(obs);
+    const numOfObservations = obsArray.length;
+    deleteObservationsList(userSession)
+      .then(() => {
+        deleteObservationFiles(userSession, numOfObservations);
+        dispatch({ type: DELETE_OBSERVATIONS });
+        return 200;
+      })
+      .catch(err => {
+        throw new Error(err.message);
```

Comment:
instead of throwing an error here, in other places we've opted to leave the `console.log` and override ESLint using `/* eslint-disable no-console */` at the top of the file

Reply:
Sure I can do that. And it depends how long it takes me to write the tests. I don't have much experience in that area and so far I have errors, but I intend to work on it today.

Reply:
Sounds good! There's some similar examples already written out so definitely use those as reference. I would say, just in the essence of time as this is pretty important to include, if you can't figure out all the tests by tonight that we merge in this PR and write out another issue for testing.

Reply:
Works for me. And thanks! I'm working off the tests that exist so hopefully I can figure out the errors.